webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/servers/PolicyServerManagementService.scala

```diff
@@ -511,7 +511,8 @@ object PolicyServerConfigurationObjects {
     )
   }
 
-  val relayTechniques = List("server-common", "rudder-service-apache","rudder-service-relayd")
+  // we need to have "common" here, even if it's not server specific, else it leads to it being deleted during migration
+  val relayTechniques = List("common", "server-common", "rudder-service-apache","rudder-service-relayd")
```

Comment:
this *might* be the cause of `common: deleted`, but I have no reproducible case for it.

Reply:
adding common here breaks migration: it will be used in the rulePolicyServer constructor, and directiveId common-${nodeId.value} , which does not exists, so it breaks